plugins/forc-index/src/ops/forc_index_stop.rs

```diff
@@ -0,0 +1,61 @@
+use crate::cli::StopCommand;
+use crate::utils::extract_manifest_fields;
+use reqwest::{
+    blocking::Client,
+    header::{HeaderMap, AUTHORIZATION},
+    StatusCode,
+};
+use serde_json::{to_string_pretty, value::Value, Map};
+use std::fs;
+use std::io::Read;
+use tracing::{error, info};
+
+pub fn init(command: StopCommand) -> anyhow::Result<()> {
+    let mut manifest_file = fs::File::open(&command.manifest).unwrap_or_else(|_| {
+        panic!(
+            "Index manifest file at '{}' does not exist",
+            command.manifest.display()
+        )
+    });
+
+    let mut manifest_contents = String::new();
+    manifest_file.read_to_string(&mut manifest_contents)?;
+    let manifest: serde_yaml::Value = serde_yaml::from_str(&manifest_contents)?;
+
+    let (namespace, identifier, _, _) = extract_manifest_fields(manifest)?;
+
+    let target = format!("{}/api/index/{}/{}", &command.url, &namespace, &identifier);
+
+    let mut headers = HeaderMap::new();
+    headers.insert(
+        AUTHORIZATION,
+        command.auth.unwrap_or_else(|| "fuel".into()).parse()?,
+    );
+
+    info!("\n🛑 Stopping index at {}", &target);
```

Comment:
Love it 👌 

Maybe `"Stopping index 'namespace.identifier' at {}"` (just being more explicit about what is being stopped)